<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ground Control System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
    <property name="spacing">
     <number>-1</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page1">
       <layout class="QVBoxLayout" name="verticalLayout_5" stretch="10,2,1">
        <item>
         <layout class="QHBoxLayout" name="layoutDashboard" stretch="2,6,2,2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1">
            <item>
             <widget class="ADI" name="dashboardADI">
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="HSI" name="dashboardHSI">
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="NAV" name="dashboardNAV">
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="PFD" name="dashboardPFD">
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustToContents</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,1">
            <item>
             <widget class="ALT" name="dashboardALT">
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="VSI" name="dashboardVSI">
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="VerticalProgress" name="motor1">
                <property name="maxValue">
                 <number>2000</number>
                </property>
                <property name="currentValue">
                 <number>0</number>
                </property>
                <property name="borderWidth">
                 <number>1</number>
                </property>
                <property name="borderRadius">
                 <number>0</number>
                </property>
                <property name="borderColorStart">
                 <color>
                  <red>0</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </property>
                <property name="borderColorEnd">
                 <color>
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </property>
               </widget>
              </item>
              <item>
               <widget class="VerticalProgress" name="motor2">
                <property name="maxValue">
                 <number>2000</number>
                </property>
                <property name="currentValue">
                 <number>0</number>
                </property>
                <property name="borderWidth">
                 <number>1</number>
                </property>
                <property name="borderRadius">
                 <number>0</number>
                </property>
                <property name="borderColorStart">
                 <color>
                  <red>0</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </property>
               </widget>
              </item>
              <item>
               <widget class="VerticalProgress" name="motor3">
                <property name="maxValue">
                 <number>2000</number>
                </property>
                <property name="currentValue">
                 <number>0</number>
                </property>
                <property name="borderWidth">
                 <number>1</number>
                </property>
                <property name="borderRadius">
                 <number>0</number>
                </property>
                <property name="borderColorStart">
                 <color>
                  <red>0</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </property>
               </widget>
              </item>
              <item>
               <widget class="VerticalProgress" name="motor4">
                <property name="maxValue">
                 <number>2000</number>
                </property>
                <property name="currentValue">
                 <number>0</number>
                </property>
                <property name="borderWidth">
                 <number>1</number>
                </property>
                <property name="borderRadius">
                 <number>0</number>
                </property>
                <property name="borderColorStart">
                 <color>
                  <red>0</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layoutData"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layoutAction" stretch="4,6">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>串口设置</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>连接</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>断开</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>IMU校准</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>启动无人机</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="pushButton_6">
              <property name="text">
               <string>停止无人机</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QWebEngineView" name="webView" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1307</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDashboard"/>
   <addaction name="actionPlanner"/>
  </widget>
  <action name="actionDashboard">
   <property name="text">
    <string>Dashboard</string>
   </property>
  </action>
  <action name="actionPlanner">
   <property name="text">
    <string>Planner</string>
   </property>
   <property name="toolTip">
    <string>Planner</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ADI</class>
   <extends>QGraphicsView</extends>
   <header>adi.h</header>
  </customwidget>
  <customwidget>
   <class>ALT</class>
   <extends>QGraphicsView</extends>
   <header>alt.h</header>
  </customwidget>
  <customwidget>
   <class>HSI</class>
   <extends>QGraphicsView</extends>
   <header>hsi.h</header>
  </customwidget>
  <customwidget>
   <class>NAV</class>
   <extends>QGraphicsView</extends>
   <header>nav.h</header>
  </customwidget>
  <customwidget>
   <class>PFD</class>
   <extends>QGraphicsView</extends>
   <header>pfd.h</header>
  </customwidget>
  <customwidget>
   <class>VSI</class>
   <extends>QGraphicsView</extends>
   <header>vsi.h</header>
  </customwidget>
  <customwidget>
   <class>VerticalProgress</class>
   <extends>QWidget</extends>
   <header>verticalprogress.h</header>
  </customwidget>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
